lib/plugins/filter/before_post_render/backtick_code_block.js

```diff
@@ -47,6 +48,14 @@ function backtickCodeBlock(data) {
             options.caption += `<a href="${match[3]}">${match[4] ? match[4] : 'link'}</a>`;
           }
         }
+
+        if (options.exclude_languages && options.exclude_languages.includes(options.lang)) {
+          // Only wrap with <pre><code class="lang"></code></pre>
+          options.wrap = false;
+          options.hljs = true;
+          options.gutter = false;
+          options.autoDetect = false;
+        }
```

Comment:
needs to be refactored into

``` js
if (options.exclude_languages) {
  if (options.exclude_language.includes(options.lang)) {
    ...
  }
}
```

I'm aware that `exclude_languages:` defaults to `[]` as shown in `default_config.js`, but `options.exclude_languages.includes()` would error if the `exclude_languages`is null,

``` yml
_config.yml
highlight:
  exclude_languages:
```

`Object.assign()` only handle non-existent key, not null value.

Reply:
Thanks for pointing this out, Fixed.